library/src/morpheus-master/bootstrap-sass-3.3.7/assets/stylesheets/bootstrap/_jumbotron.scss

```diff
@@ -17,8 +17,8 @@
 
   p {
     margin-bottom: ($jumbotron-padding / 2);
-    font-size: $jumbotron-font-size;
-    font-weight: 200;
+    font-size: $default-font-size;
```

Comment:
Now "$jumbotron-font-size" variable is unused, maybe its new value should be $default-font-size?